src/components/zoomableImage/index.scss

```diff
@@ -34,7 +34,8 @@
 .zoomable__toggle:focus + .zoomable__container {
   @include focus-border;
 }
-.zoomable__toggle:hover + .zoomable__container {
+.zoomable__toggle:hover + .zoomable__container,
+.zoomable__container:hover {
```

Comment:
Good find, I actually think `zoomable__container` should be the only with the hover-border, since it's the one that takes up the most amount of space with its padding.